README.rst

```diff
@@ -273,6 +277,10 @@ These options are available for the recipe part plone.recipe.varnish:configurati
     clients. Useful if there is another port mapping in front of varnish, such
     as haproxy. Defaults to 443.
 
+``vcl-version``
+    Varnish VCL format version.
+    If not given it defaults to ``4.0``.
```

Comment:
So we have varnish 5.2 as default version, but 4.0 as default vcl-version. Is that correct? Seems strange at first sight. If it is correct, you may want to explain a bit, maybe say which varnish versions are compatible with which vcl-versions.

Reply:
Yes :) i'm look here and fixing

About VCL version, although the Varnish version's 4 or 5, we are talking about the VCL syntax version (https://book.varnish-software.com/4.0/chapters/VCL_Basics.html#vcl-syntax).

I'm adding this parameter (vcl-version) foreseeing a near future, so, when the VCL syntax version has a new release, we can say which one we want to use.